--- gnupg-1.9.17/g10/misc.c.orig	2005-06-15 17:34:29.000000000 +0200
+++ gnupg-1.9.17/g10/misc.c	2005-06-23 23:04:14.288425848 +0200
@@ -989,7 +989,7 @@
 	char *buffer;
 
 	rc = gcry_mpi_aprint( GCRYMPI_FMT_HEX,
-                              &(unsigned char*)buffer, NULL, a );
+                              (unsigned char**)&buffer, NULL, a );
 	assert( !rc );
 	fputs( buffer, fp );
 	n += strlen(buffer);
