--- gnupg-2.5.18/agent/command-ssh.c.orig	2025-11-19 16:55:57.000000000 +0100
+++ gnupg-2.5.18/agent/command-ssh.c	2026-04-01 23:30:52.453206379 +0200
@@ -1481,7 +1481,7 @@
       err = gcry_mpi_print (GCRYMPI_FMT_USG, data, spec->keysize, &data_n, s);
       if (data_n < spec->keysize)
         {
-          memmove (data, data+spec->keysize-data_n, data_n);
+          memmove (data+spec->keysize-data_n, data, data_n);
           memset (data, 0, spec->keysize-data_n);
           data_n = spec->keysize;
         }
